app/components/Settings/Preview/components/getPlotJsonFromState.js

```diff
@@ -142,6 +145,8 @@ export default function getPlotJsonFromState(state) {
             }
         }
 
+        // TODO should an .includes(traceType) be added?
+        // eslint-disable-next-line no-constant-condition
         if (['scattergeo-world', 'choropleth-world', 'scattergeo-usa', 'choropleth-usa']) {
```

Comment:
yeah, I'm not sure what's going on here - @jackparmer ?

Reply:
but my guess is that this should be `if (R.includes(traceType, ['scattergeo-world', 'choropleth-world', 'scattergeo-usa', 'choropleth-usa'])`

Reply:
yep, sorry about that, `if (R.includes(traceType, ['scattergeo-world', 'choropleth-world', 'scattergeo-usa', 'choropleth-usa'])` is the correct intention